src/oauth2_backend.erl

```diff
@@ -112,18 +112,6 @@
 -callback verify_scope(scope(), scope(), appctx()) ->
                     {ok, {appctx(), scope()}} | {error, notfound | badscope}.
 
-%% @doc Sign the grant context with a private key and produce a JWT.
-%%      The grant context is a proplist carrying information about the identity
-%%      with which the token is associated, when it expires, etc.
--callback jwt_sign(grantctx(), appctx()) -> {ok, token()}.
-
-%% @doc Verifies a JWT, returning the corresponding grant context if
-%%      verification succeeds.
--callback jwt_verify(token()) -> {ok, grantctx()} | {error, badjwt}.
-
-%% @doc A case-sensitive string or URI that uniquely identifies the issuer.
--callback jwt_issuer() -> binary().
-
```

Comment:
These call backs were not going to be removed because it's a public repo? Commenting so we don't forget.